tests/unit/log/test_http_header_caller_retrieval.py

```diff
@@ -13,12 +14,11 @@
 
 
 class ActionLogTest(unittest.TestCase):
-
     def test(self) -> None:
-        with app.test_request_context(), patch.object(http_header_caller_retrieval, 'request') as mock_request:
+        with app.test_request_context(), \
```

Comment:
I am very confused why this was working previously. `patch.object` returns an AsyncMock by default (see https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch) and that was not being used correctly.